profiles/hooks/addCurrentProfile.js

```diff
@@ -0,0 +1,17 @@
+const merge = require('ramda/src/merge')
+const isNil = require('ramda/src/isNil')
+const path = require('ramda/src/path')
+
+module.exports = function addCurrentProfile (hook) {
+  if (isNil(path(['params', 'credential', 'agentId'], hook))) return hook
+
+  const profiles = hook.app.service('profiles')
+
+  return profiles.find({ query: { agentId: hook.params.credential.agentId } })
+    .then((profiles) => {
+      hook.params = merge(hook.params, {
```

Comment:
i think we should assume the current agent has only one profile. at least for now. so `params.profile`. what do you think?